Airports/EGDR/Positions.txt

```diff
@@ -0,0 +1,3 @@
+EGDR_APP:Culdrose Approach:134.050:DRA:A:EGDR:APP:-:-:7030:7045:N050.05.09:W005.15.17::
+EGDR_P_APP:Culdrose Talkdown:123.300:DRP:P:EGDR:APP:-:-:7030:7045:N050.05.09:W005.15.17::
+EGDR_TWR:Culdrose Tower:122.100:DRT:T:EGDR:TWR:-:-:7030:7045:N050.05.09:W005.15.17::
```

Comment:
Same frequency as Predannack? Will cause issues with ES text if both are online at the same time due to their close proximity

Reply:
122.100 a NATO common tower frequency that is used as a backup for UHF. It's the same in a lot of places, not sure how to get around this issue. It also applies to lots of the talkdown frequencies.